src/provider/qti.js

```diff
@@ -317,7 +318,7 @@ var qtiProvider = {
                         // when the test part is linear, the item is always answered as we cannot come back to it
                         const testPart = self.getCurrentPart();
                         const isLinear = testPart && testPart.isLinear;
-                        currentItem.answered = isLinear || currentItemHelper.isAnswered(self);
+                        currentItem.answered = isLinear || currentItemHelper.isAnswered(self, partiallyAnsweredIsAnswered);
```

Comment:
How do you think, should it influence `"Allow skipping"` validation? Now it allows me to skip partially answered item. I see that customer doesn't mention it in the ticket , so maybe this is completely fine.

Reply:
> Now it allows me to skip partially answered item

What do you mean by this? When skipping an item, it does not matter if it is answered, partially or completely, or not. The answers are set to the default value and that's it.

If you mean the item can be submitted partially answered, yes this is the expected behavior when the option is set.

Reply:
I mean, "_A response to this item is required_" dialog does NOT appear if item is answered partially. 
But now I noticed that it already has its own setting for this: `pluginConfig.allowPartial` https://github.com/oat-sa/tao-test-runner-qti-fe/blob/dc4793d0407db480941c3d6e37da94e6cf7899a2/src/plugins/navigation/allowSkipping.js#L89
So this can be closed.